core/src/main/java/org/apache/accumulo/core/client/impl/ConnectorImpl.java

```diff
@@ -193,4 +221,151 @@ public synchronized ReplicationOperations replicationOperations() {
 
     return replicationops;
   }
+
+  public static class ConnectorBuilderImpl implements InstanceArgs, PropertyOptions, ConnectionInfoOptions, AuthenticationArgs, ConnectionOptions, SslOptions,
+      SaslOptions, ConnectorFactory {
+
+    private ConnectionInfoImpl info = new ConnectionInfoImpl();
```

Comment:
Shouldn't this type be ConnectionInfo?

Reply:
In 705d604759f9495cb, I changed the builder to create a `Properties` object